src/components/containers/ContainersListContent.js

```diff
@@ -12,28 +12,36 @@ import {list, listTemplateActions} from "../../modules/containers/actions";
 import {actionsToButtonList} from "../../utils/templateActions";
 
 const TABLE_COLUMNS = [
+  {
+    title: "Name",
+    dataIndex: "name",
+  },
   {
     title: <><BarcodeOutlined style={{marginRight: "8px"}} /> Barcode</>,
     dataIndex: "barcode",
     render: (barcode, container) => <Link to={`/containers/${container.id}`}>{barcode}</Link>,
   },
   {
-    title: "Name",
-    dataIndex: "name",
+    title: "Sample",
+    dataIndex: "samples",
+    render: (samples, container) =>
+      (container.kind == "tube" && samples.length > 0) ? 
+        <Link to={`/samples/${samples[0].id}`}>{samples[0].name}</Link> : 
+        null,
```

Comment:
Oh also: why is only the first sample used?

Reply:
Thanks for the feedback Romain ... Yeah i went this way because i spent a lot of time trying to figure a way to do without breaking the way it was currently working and thought at some point i would be better with something that worked althought in a less elegant manner. I'll get to reading the links you sent and go back to work to use normalized objects.

As for the question of the tube it's been asked specifically by Alexandre in his analysis. The list do not really have the space to receive 96 samples from a plate... So i need to make sure only the tube that have a single sample get shown... That is why i only get the first... i do not expect more since it is a tube. I could iterate through the list anyway but it would augment the chance that down the line we remove the tube limitation and get all the samples in the lists.